context/defaultSkuConfig.js

```diff
@@ -48,4 +48,6 @@ module.exports = {
   orderImports: false,
   cspEnabled: false,
   cspExtraScriptSrcHosts: [],
+  useHttpsDevServer: false,
+  devServerMiddleware: null,
```

Comment:
Should this be `dev-middleware.js`? I don't anywhere else that it does the defaulting.

Reply:
I'd like them to manually specify it for it to be activated.

Reply:
> Place a `dev-middleware.js` file in the root of the project. The file must export a function that will receive the express server.

Docs seems to say that's the default file though. Maybe just make the docs clearer that you need to supply a path to a file then.